include/service/entities/user_service.js

```diff
@@ -136,31 +136,50 @@ module.exports = function UserServiceModule(pb) {
 
         return adminOptions;
     };
-
+    
     /**
      * Retrieves a select list (id/name) of available system editors
+     * @deprecated since 0.4.0
      * @method getEditorSelectList
      * @param {String} currId The Id to be excluded from the list.
      * @param {Function} cb A callback that takes two parameters.  The first is an
      * error, if exists, the second is an array of objects that represent the
      * editor select list.
      */
-    UserService.prototype.getEditorSelectList = function(currId, cb) {
+    UserService.prototype.getEditorSelectList(currId, cb) {
+        pb.log.warn('UserService: getEditorSelectList is deprecated. Use getWriterOrEditorSelectList instead');
+        this.getWriterOrEditorSelectList(currId, cb);
+    };
+
+    /**
+     * Retrieves a select list (id/name) of available system writers or editors
+     * @method getWriterOrEditorSelectList
+     * @param {String} currId The Id to be excluded from the list.
+     * @param {Boolean} [getWriters=false] Whether to retrieve all writers or just editors.
+     * @param {Function} cb A callback that takes two parameters.  The first is an
+     * error, if exists, the second is an array of objects that represent the
+     * editor select list.
+     */
+    UserService.prototype.getWriterOrEditorSelectList = function(currId, getWriters, cb) {
+        if (util.isFunction(getWriters)) {
+            cb = getWriters;
+            getWriters = false;
+        }
+        
         var self = this;
 
         var opts = {
             select: {
-                _id: 1,
```

Comment:
I removed the reference to ```_id``` for two reasons.  

1) It is implicitly implied that the id field will be returned by the select 
2) I hate the leading underscore